### ISSUE TITLE
Applying some functionalities of CIF to my own data

### ISSUE THREAD

**Issue by shahmeer99** [LenkaV/CIF#25]
If I have a pandas DF of time series data in which each column corresponds to an individual indicator and each row has the monthly values. How would I go about applying the following steps mentioned in the README:
1. data transformations (seasonal adjustment, stabilising forecasts, detrending, normalization)
2. ex-post turning points detection (Bry-Boschan algorithm)
3. aggregation into composite indicator

I tried following the example notebook given on the repo but couldn't figure out the necessary preprocessing and library commands required to do this. Would appreciate any help!

**Comment by LenkaV** [comment]
Your question is rather generic. Can you download the example data as shown in the notebook and compare it with your data set?

**Comment by shahmeer99** [comment]
Yeah I'm sorry for the lack of clarity, its just that my econ knowledge is fairly limited. For my task, I have data which looks like shown image (many more columns than shown), and I want to basically apply the functionalities/steps mentioned in my initial comment to it. The input is that data, the output should be an "index value". Looking at the dataset format in the notebook, there are measures that have specific names ("STSA", "CXML") and subjects named "BCBUTE02", "BREMFT02" etc. I don't understand that format and thus can't transform my data accordingly. Is it possible to do my task given the data I have using this package?

<img width="1288" alt="Screen Shot 2023-04-23 at 2 58 55 AM" src="https://user-images.githubusercontent.com/46823325/233812503-0781bee6-9870-44b9-b3a5-3c9c3434f62c.png">
